persistence/trees/module.go

```diff
@@ -0,0 +1,73 @@
+package trees
+
+import (
+	"fmt"
+
+	"github.com/pokt-network/pocket/persistence/kvstore"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/smt"
+)
+
+func (*treeStore) Create(bus modules.Bus, options ...modules.TreeStoreOption) (modules.TreeStoreModule, error) {
+	m := &treeStore{}
+
+	for _, option := range options {
+		option(m)
+	}
+
+	m.SetBus(bus)
```

Comment:
```suggestion
	bus.RegisterModule(m)
```

Is there a difference in these calls?

Reply:
@bryanchriswhite would be able to explain the difference better. I just followed his pointers on how to get my module working.